### ISSUE TITLE
Accessibility: Cannot open or close "help" information

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-campaign-admin#366]
In campaign admin, there's a `?` button with the aria label "Show help" which when clicked displays additional information. When the information is displayed, the `?` button is replaced with a `x` with aria label "Hide help".

Both of these buttons _are_ in the tab focus order, but:
1. There is no visible focus state
2. When they have focus, neither the enter nor space keys do anything
3. They're both `<a>` tags but should be `<button>`.

## Acceptance criteria

- [ ] Appropriate `:focus-visible` CSS is applied (see https://github.com/silverstripe/.github/issues/443)
- [x] Pressing `space` and `enter` keys is the same as clicking the button.
- [x] `<a>` tag is replaced with `<button>` tag
- [x] The changelog clearly calls out any changes to markup selectors so people can adjust their custom css/js as necessary

## PRs
- https://github.com/silverstripe/silverstripe-campaign-admin/pull/371
- https://github.com/silverstripe/developer-docs/pull/819

**Comment by GuySartorelli** [comment]
PRs merged
